### ISSUE TITLE
[Feature]: Using song timers as seek buttons

### ISSUE THREAD

**Issue by NikunjKhangwal** [fast4x/RiMusic#1082]
### Description

Currently seeking a song 5 second back or ahead is a bit inconvenient because you have to hold the back and previous button to do it.

### Suggested Solution

Increase the size (or touch area) of these button and use them to seek instead of previous and back buttons. These two doesn't serve any other purpose and doesn't demand extra space. Also it would be easier to use because now you can just tap them to seek instead of holding like you have to do now.
![IMG_20240316_123011](https://github.com/fast4x/RiMusic/assets/99175083/79c011d9-ebb8-4e4a-b185-79210f44888b)

### Alternatives

Or you can use the traditional way other music players are doing it which is to add additional seeking buttons. This obviously require extra space on the screen. Both of these solutions work for me. You can choose which one you like better or which one is less time consuming and easier to implement.
![IMG_20240316_122259](https://github.com/fast4x/RiMusic/assets/99175083/5fbf2ebe-73b0-4859-bb89-6d105eccf906)

**Comment by SureshTimma** [comment]
It's better to add that feature to the thumbnail. 

When we double tap right side on thumbnail the song should forward for 10 seconds. I think it's better idea

**Comment by NikunjKhangwal** [comment]
> It's better to add that feature to the thumbnail.
> 
> When we double tap right side on thumbnail the song should forward for 10 seconds. I think it's better idea

Yeah this works as well, didn't crossed my mind. Currently double tapping on cover art is used to add/remove songs in favorites. It can be changed so the song gets added to favourites when you double in middle of cover art. Keeping left and right side for seeking.

**Comment by dAtA-TRoN** [comment]
> It's better to add that feature to the thumbnail.\n\nWhen we double tap right side on thumbnail the song should seek forward for 10 seconds. I think it's better idea

Perfect solution 👏🏼😊!

**Comment by dAtA-TRoN** [comment]
> It can be changed so the song gets added to favourites when you double in middle of cover art.

I think that would be too cluttered. There's a ♥️ button for that, which is also easily accessible.

**Comment by NikunjKhangwal** [comment]
> I think that would be too cluttered. There's a ♥️ button for that, which is also easily accessible.

Ok i forgot about that too 💀

**Comment by NikunjKhangwal** [comment]
The amounts of dislike lol. Ok guys it was a bad idea. 🥲

**Comment by dAtA-TRoN** [comment]
> The amounts of dislike lol. Ok guys it was a bad idea. 🥲

It's okay, we all have shitty ideas sometimes 🤷🏻‍♂️.
